recipe/meta.yaml

```diff
@@ -43,26 +43,26 @@ about:
   summary: John Snow Labs provides a library for delivering safe & effective NLP models.
   license: Apache-2.0
   license_file: LICENSE
+  license_url: https://github.com/JohnSnowLabs/nlptest/blob/main/LICENSE
   description: |
     `nlptest` is an open-source Python library designed to help developers
     deliver safe and effective Natural Language Processing (NLP) models.
 
     PyPI: [https://pypi.org/project/nlptest/](https://pypi.org/project/nlptest/)
 
     ---
-
     :fire: The conda-forge recipe was generated with [Conda-Forger App](https://sugatoray-conda-forger.streamlit.app/).
-
+    
     :point_right: [![Streamlit App](https://static.streamlit.io/badges/streamlit_badge_black_white.svg)](https://sugatoray-conda-forger.streamlit.app/)
-
     [_streamlit-conda-forger-app]: https://sugatoray-conda-forger.streamlit.app/
-
+    
   dev_url: https://github.com/JohnSnowLabs/nlptest/
   doc_url: https://nlptest.org/docs/pages/docs/install
 
 extra:
   recipe-maintainers:
     - sugatoray
     # maintainers from upstream
-    - johnsnowlabs
+    - johnsnowlab
```

Comment:
As far as I know this has to be a GitHub user. 

@johnsnowlabs is a GitHub user (owner of the upstream library) and `johnsnowlab` is possibly the anaconda.org user ID you use. That ID cannot be user here. 

Please confirm that @johnSnowLab is valid GitHub user.

Reply:
Yes that's the correct GH user, the previous one you had is the GH organization, not a user